### PR TITLE
IDEX-3768: Add ssh widget to console panel

### PR DESCRIPTION
**Pull request by vzhukovskii**
@vparfonov @ashumilova @slemeur

### CODE REVIEW

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 1]
```diff
@@ -92,6 +100,50 @@
...
+            return null;
+        }
+
+        @Override
+        public String getCommandTemplate() {
+            return "";
+        }
+
+        @Override
+        public String getPreviewUrlTemplate() {
```
Comment (vparfonov) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: PreviewUrl ?

Comment (vzhukovskii) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: It's client command type configuration, we need this method to provide default preview template for each command type.

**Comment by vparfonov** [comment]
ok

**Comment by ashumilova** [comment]
+1

File: plugin-machine/che-plugin-machine-ext-client/src/main/resources/org/eclipse/che/ide/extension/machine/client/MachineLocalizationConstant.properties [hunk 2]
```diff
@@ -150,3 +150,6 @@ failed.to.create.recipe=Failed to create recipe
 failed.to.save.recipe=Failed to save recipe
 failed.to.find.machine=Failed to find machine {0}
 failed.to.get.processes=Failed to get processes from machine {0}
+
+ssh.connect.info=You can connect your SSH client to the machine ''{0}'' by this command:\
+  \nssh user@{1} -p {2} [-i /path/to/your/ssh]
```
Comment (sleshchenko) [inline plugin-machine/che-plugin-machine-ext-client/src/main/resources/org/eclipse/che/ide/extension/machine/client/MachineLocalizationConstant.properties]: I think we should write something like [-i /path/to/your/ssh/key] or [-i /path/to/your/key]

Comment (vzhukovskii) [inline plugin-machine/che-plugin-machine-ext-client/src/main/resources/org/eclipse/che/ide/extension/machine/client/MachineLocalizationConstant.properties]: Selected first variant

**Comment by slemeur** [comment]
+1. Fine for me.

**Pull request merged**